tests/FakeItEasy.Tests.TestHelpers/StringExtensions.cs

```diff
@@ -0,0 +1,27 @@
+namespace FakeItEasy.Tests.TestHelpers
+{
+    using System;
+    using System.Text.RegularExpressions;
+
+    public static class StringExtensions
+    {
+        public static string NormalizeLineEndings(this string s)
+        {
+            if (string.IsNullOrEmpty(s))
+            {
+                return s;
+            }
+
+#if FEATURE_STRING_CONTAINS_COMPARISONTYPE
+            if (!s.Contains('\n', StringComparison.Ordinal))
+#else
+            if (s.IndexOf('\n') < 0)

```

Comment:
I don't use this on .NET Standard 2.1, because it causes a warning. But now that I think of it, maybe we don't need a .NET Standard 2.1 target for the test helpers...

Reply:
Ah, I just remembered why we needed it... It references FakeItEasy, which is only built on netstandard2.1 in the netcore3.0 profile.

Reply:
Yes, because in netstandard2.1 there's an overload that takes a `StringComparison`, which should be used instead